### DISCUSSION TITLE
How can I change StandardFontFamily in WebView2?

### DISCUSSION THREAD

**Discussion by willowTank** [MicrosoftEdge/WebView2Feedback#3908] [Q&A]
Hello!
I am using Webview in C# .net application.
I want to change StandardFontFamily of browser. 
Is there any way for me?
Thank you

**Comment by xblack0** [comment]
* {
    font-family: fonthere 
}
as css > 
is this will solve your issue or ur looking for webivew2 control font ?

**Comment by willowTank** [comment]
Hi @xblack0 
thank you for your reply.
I want to change the default font of webview2 because I can avoid detection of fingerprint.
